### PR TITLE
Chore: storybook gluestack story examples

### PR DESCRIPTION
**Pull request by GerganaNIvanova**
## Description

Included example stories using gluestack components

## Related Issue

- N/A

## Proposed Changes

- Added example stories using gluestack components for the case when a user decides to use storybook with gluestack-ui
- Added mobile specific scripts for generating and watching stories and updated the README file accordingly 

## Screenshots (if appropriate)

Please attach any screenshots or images if they help in understanding the changes.

## Checklist

- [x] I have performed a self-review of my code.
- [x] My code follows the project's coding standards.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding CLI changes to the documentation.
- [x] My changes generate no new warnings or errors.
- [x] I have tested the CLI with my changes locally.
- [x] baseProject is fully functional on both Android & iOS.

## Additional Information

Any additional information or context that could be relevant to the reviewer.

### CODE REVIEW

File: baseProject/config/storybook/gluestackExamples/stories/Button.stories.js
Comment (NikolayDonev) [inline baseProject/config/storybook/gluestackExamples/stories/Button.stories.js:1]: @GerganaNIvanova this file needs to be TS

**Review by NikolayDonev — CHANGES_REQUESTED** [review]

File: baseProject/config/storybook/gluestackExamples/stories/Button.stories.js
Comment (GerganaNIvanova) [reply baseProject/config/storybook/gluestackExamples/stories/Button.stories.js:1]: Done!

File: src/commands/start-project.ts [hunk 1]
```diff
@@ -177,12 +181,26 @@ const startProject = async (toolbox: RnBootstrapToolbox) => {
...
+      // Replace storybook web files with preconfigured ones specific for gluestack
       replaceFile('preview.tsx', '.storybook/preview.ts');
       filesystem.rename(
         `${process.cwd()}/.storybook/preview.ts`,
         'preview.tsx'
       );
+      // Remove files with specified extensions from the 'src/stories' folder
+      const extensionsToRemove = ['ts', 'tsx', 'css'];
+      const files = filesystem.list('src/stories');
+      files
```
Comment (NikolayDonev) [inline src/commands/start-project.ts:193]: @GerganaNIvanova let's define this constant as stories and filter them as storyFiles.

Comment (GerganaNIvanova) [reply src/commands/start-project.ts]: Done!

**Review by NikolayDonev — APPROVED** [review]

**Pull request merged**